sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_create_composed_model.py

```diff
@@ -0,0 +1,117 @@
+    SAS URLs for this sample.
+
+USAGE:
+    python sample_create_composed_model.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+    3) PURCHASE_ORDER_OFFICE_SUPPLIES_SAS_URL - a container SAS URL to your Azure Storage blob container.
+    4) PURCHASE_ORDER_OFFICE_EQUIPMENT_SAS_URL - a container SAS URL to your Azure Storage blob container.
+    5) PURCHASE_ORDER_OFFICE_FURNITURE_SAS_URL - a container SAS URL to your Azure Storage blob container.
+    6) PURCHASE_ORDER_OFFICE_CLEANING_SUPPLIES_SAS_URL - a container SAS URL to your Azure Storage blob container.
+"""
+
+import os
+
+
+class ComposedModelSample(object):
+
+    def create_composed_model(self):
+        # [START begin_create_composed_model]
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer import FormTrainingClient
+
+        endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+        key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+        po_supplies = os.environ['PURCHASE_ORDER_OFFICE_SUPPLIES_SAS_URL']
+        po_equipment = os.environ['PURCHASE_ORDER_OFFICE_EQUIPMENT_SAS_URL']
+        po_furniture = os.environ['PURCHASE_ORDER_OFFICE_FURNITURE_SAS_URL']
+        po_cleaning_supplies = os.environ['PURCHASE_ORDER_OFFICE_CLEANING_SUPPLIES_SAS_URL']
+
+        form_training_client = FormTrainingClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+        supplies_poller = form_training_client.begin_training(
+            po_supplies, use_training_labels=True, display_name="Purchase order - Office supplies"
+        )
+        equipment_poller = form_training_client.begin_training(
+            po_equipment, use_training_labels=True, display_name="Purchase order - Office Equipment"
+        )
+        furniture_poller = form_training_client.begin_training(
+            po_furniture, use_training_labels=True, display_name="Purchase order - Furniture"
+        )
+        cleaning_supplies_poller = form_training_client.begin_training(
+            po_cleaning_supplies, use_training_labels=True, display_name="Purchase order - Cleaning Supplies"
+        )
+        supplies_model = supplies_poller.result()
+        equipment_model = equipment_poller.result()
+        furniture_model = furniture_poller.result()
+        cleaning_supplies_model = cleaning_supplies_poller.result()
+
+        models_trained_with_labels = [
+            supplies_model.model_id,
+            equipment_model.model_id,
+            furniture_model.model_id,
+            cleaning_supplies_model.model_id
+        ]
+
+        poller = form_training_client.begin_create_composed_model(
+            models_trained_with_labels, display_name="Office Supplies Composed Model"
+        )
+        model = poller.result()
+
+        print("Purchase Order Model Info:")
```

Comment:
Should this be `Office Supplies Composed Model Info:` ?

Reply:
sure!